generators/advanced/templates/root/_.gitignore

```diff
@@ -0,0 +1,3 @@
+node_modules
```

Comment:
Maybe we can extend the rules a little using [a generator](http://gitignore.io/api/node,code,linux,macos,windows,webstorm,sublimetext)

Reply:
right new [issue](https://github.com/guidesmiths/generator-systemic/issues/52)